gdal/ogr/ogrsf_frmts/amigocloud/ogr_amigocloud.h

```diff
@@ -44,6 +44,7 @@
 json_object* OGRAMIGOCLOUDGetSingleRow(json_object* poObj);
 CPLString OGRAMIGOCLOUDEscapeIdentifier(const char* pszStr);
 CPLString OGRAMIGOCLOUDEscapeLiteral(const char* pszStr);
+std::string json_encode(const std::string &value);
```

Comment:
As json_encode is a pretty generic name, please use some namespace/prefixing mechanism to avoid potentially symbol clash with other parts of GDAL or code using it